v2.1/configure-replication-zones.md

```diff
@@ -105,425 +95,58 @@ Constraints can be specified such that they apply to all replicas in a zone or s
-`rm` | Remove a replication zone.
-
-## Synopsis
-
-~~~ shell
-# List all replication zones:
-$ cockroach zone ls <flags>
-
-# View the default replication zone for the cluster:
-$ cockroach zone get .default <flags>
-
-# View the replication zone for a database:
-$ cockroach zone get <database> <flags>
-
-# View the replication zone for a table:
-$ cockroach zone get <database.table> <flags>
-
-# View the replication zone for an index:
-$ cockroach zone get <database.table@index> <flags>
-
-# View the replication zone for a table or index partition:
-$ cockroach zone get <database.table.partition> <flags>
-
-# Edit the default replication zone for the cluster:
-$ cockroach zone set .default --file=<zone-content.yaml> <flags>
-
-# Create/edit the replication zone for a database:
-$ cockroach zone set <database> --file=<zone-conent.yaml> <flags>
-
-# Create/edit the replication zone for a table:
-$ cockroach zone set <database.table> --file=<zone-content.yaml> <flags>
-
-# Create/edit the replication zone for an index:
-$ cockroach zone set <database.table@index> --file=<zone-content.yaml> <flags>
-
-# Create/edit the replication zone for a table or index partition:
-$ cockroach zone set <database.table.partition> --file=<zone-content.yaml> <flags>
-
-# Remove the replication zone for a database:
-$ cockroach zone rm <database> <flags>
-
-# Remove the replication zone for a table:
-$ cockroach zone rm <database.table> <flags>
-
-# Remove the replication zone for an index:
-$ cockroach zone rm <database.table@index> <flags>
-
-# Remove the replication zone for a table or index partition:
-$ cockroach zone set <database.table.partition> --file=<zone-content.yaml> <flags>
-
-# View help:
-$ cockroach zone --help
-$ cockroach zone ls --help
-$ cockroach zone get --help
-$ cockroach zone set --help
-$ cockroach zone rm --help
-~~~
-
-## Flags
-
-The `zone` command and subcommands support the following [general-use](#general) and [logging](#logging) flags.
-
-### General
-
-Flag | Description
------|------------
-`--disable-replication` | Disable replication in the zone by setting the zone's replica count to 1. This is equivalent to setting `num_replicas: 1`.
-`--echo-sql` | Reveal the SQL statements sent implicitly by the command-line utility. For a demonstration, see the [example](#reveal-the-sql-statements-sent-implicitly-by-the-command-line-utility) below.
-`--file`<br>`-f` | The path to the [YAML file](#replication-zone-format) defining the zone configuration. To pass the zone configuration via the standard input, set this flag to `-`.<br><br>This flag is relevant only for the `set` subcommand.
-
-### Client connection
-
-{% include {{ page.version.version }}/sql/connection-parameters.md %}
+## View replication zones
 
-See [Client Connection Parameters](connection-parameters.html) for more details.
+Use the [`SHOW ZONE CONFIGURATIONS`](show-zone-configurations.html) statement to view details about existing replication zones.
```

Comment:
It looks like this is linking away, but a couple of paragraphs down we show how to do the same thing (IIUC?).  A pattern I've sometimes used in situations like this is to link from a section like this down to the example further down the page.  From the example, I might then say something like "here's how you do X.  For more information, see X REFERENCE PAGE".  I think something like that could work here.  The benefit IMO is that the reader is not jumping to another page to get the basic info, but they can if they need more details.  Your call of course.